### ISSUE TITLE
The example BasicRegionGrowing fails to compile

### ISSUE THREAD

**Issue by yurivict** [InsightSoftwareConsortium/ITKSphinxExamples#400]

**Comment by dzenanz** [comment]
TL;DR This example is broken.

This example [isn't enabled](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/blob/v5.2.0/src/Segmentation/KLMRegionGrowing/CMakeLists.txt) when I configure the whole repository. The likely reason is that it doesn't build, and from the looks of git history it has been broken ever since it was migrated from the Wiki Examples. The error message on my computer is `3>C:\Dev\ITKExamples\src\Segmentation\KLMRegionGrowing\BasicRegionGrowing\Code.cxx(24,10): fatal  error C1083: Cannot open include file: 'vtkImageViewer.h': No such file or directory`, but I also see squiggles under `SetLower` etc. By looking at the code of `itk::RegionGrowImageFilter`, it is an abstract class.

**Comment by dzenanz** [comment]
While looking into this, I uncovered https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/pull/401.
